### ISSUE TITLE
Update Column Selectors

### ISSUE THREAD

**Issue by amycoffin** [BRCAChallenge/brca-exchange#1001]
- [ ] Add column selector for In Silico Prior Probabilities (overall prior displayed)
- [ ] Add column selector for Literature Search results (number of papers displayed in column) 
- [ ] Add intron/exon number column selector and/or filter (to be approved) 
- [ ] Add Functional Assay Results (or Findlay Function Scores?) column selector (displays function score)

**Comment by amycoffin** [comment]
@zfisch I'd like this one to stay open, because I think some meaningful work can be done on filters, but ultimately it's very low priority. Not until users ask for better filtering will this become relevant. It'd be nice if we empowered users to custom-filter our data to streamline their research, but its pie-in-the-sky for now.
